src/executor.cc

```diff
@@ -30,14 +30,16 @@ Executor::Executor() = default;
 Executor::~Executor() = default;
 
 Result Executor::CompileShaders(const amber::Script* script,
-                                const ShaderMap& shader_map) {
+                                const ShaderMap& shader_map,
+                                bool disable_spirv_validation) {
   for (auto& pipeline : script->GetPipelines()) {
     for (auto& shader_info : pipeline->GetShaders()) {
       ShaderCompiler sc(script->GetSpvTargetEnv());
 
       Result r;
       std::vector<uint32_t> data;
-      std::tie(r, data) = sc.Compile(&shader_info, shader_map);
+      std::tie(r, data) =
+          sc.Compile(&shader_info, shader_map, disable_spirv_validation);
```

Comment:
Let's pass disable_spirv_validation into the ShaderCompiler constructor instead since it will be global to the compiler

Reply:
Done.